writeup/gz_hubble_data.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[twocolumn]{aastex}
+\documentclass[twocolumn]{aastex6}
```

Comment:
Odd. Have you tried modifying the aastex6.cls file to change `aastex` -> `aastex6`?